### DISCUSSION TITLE
Spectrum analyzer link broken

### DISCUSSION THREAD

**Discussion by oisact** [AubsUK/uv-k5-firmware-custom#12] [General]
The link in this repo (and egzumer's) to the fagci spectrum analyzer is broken / 404. That contained the instructions on how to use it, which is somewhat opaque and complex process.  I can't remember them offhand. It might be good to put the key commands directly in your readme.

**Comment by AubsUK** [comment]
In Egzumer's [Wiki](https://github.com/egzumer/uv-k5-firmware-custom/wiki), there's a section for [Spectrum-analyzer](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Spectrum-analyzer).
